bazel/api-gen/rendering/transforms/code-transforms.ts

```diff
@@ -313,7 +307,20 @@ function isOptionalMember(member: PropertyEntry): boolean {
 
 function getTags(member: PropertyEntry): string[] {
   return member.memberTags
-    .map((tag) => ANGULAR_PROPERTY_DECORATORS[tag] ?? tag)
+    .map((tag) => {
+      if (tag === 'input') {
+        !member.inputAlias || member.name === member.inputAlias
```

Comment:
Should this be 
```suggestion
        return !member.inputAlias || member.name === member.inputAlias
```


I think as written we don't return any string at all on most paths through this function, maybe I am confused at what you are trying to do here

Reply:
Good catch !